### DISCUSSION TITLE
Viewed icon issues

### DISCUSSION THREAD

**Discussion by MrRoiz** [pwntester/octo.nvim#383] [General]
I reinstalled Ubuntu on my PC and using my exactly same config as before viewed icons now are the same as the file type
![image](https://user-images.githubusercontent.com/49358353/227578587-0d1b4049-b9ed-4332-801a-619f47af6734.png)

- Using a nerd font (same as before)

Not sure if I missed something, that's why I put this on discussions instead of issues

**Comment by williambdean** [comment]
Hi @MrRoiz, any luck with this? Are you still experiencing this?
